src/ontology/mondo-ingest.Makefile

```diff
@@ -598,24 +599,26 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
 	--onto-config-path metadata/$*.yml
 
+# Synchronization: Synonyms
 # todo: -deleted case to be added eventually
-# todo: when done with development, remove 'tmp/synonym_sync_combined_cases.tsv' and replace goal below
 .PHONY: sync-synonyms
-sync-synonyms: tmp/synonym_sync_combined_cases.tsv
-#sync-synonyms: $(REPORTDIR)/sync-synonyms.added.tsv $(REPORTDIR)/sync-synonyms.confirmed.tsv $(REPORTDIR)/sync-synonyms.updated.tsv
+sync-synonyms: tmp/synonym_sync_combined_cases.tsv $(REPORTDIR)/sync-synonyms.added.tsv $(REPORTDIR)/sync-synonyms.confirmed.tsv $(REPORTDIR)/sync-synonyms.updated.tsv
```

Comment:
There was no discussion about `$(REPORTDIR)/%-synonyms.deleted.robot.tsv`.

Reply:
@matentzn We all decided we are not doing the 'deleted' case yet. It was more complicated than we originally imagined.

That `tmp/` file is the google sheet everything concatenated together. It's the google sheet we are using. If that shouldn't be a tmp file, let me know. If it should be, then this is here just temporarily (there is a `todo` comment above it about that) so that it's easier for me to run.